internal/cli/test.go

```diff
@@ -193,7 +193,7 @@ func testTokenCmd(cli *cli) *cobra.Command {
 				return err
 			}
 
-			if err := testAudience.Ask(cmd, &inputs.Audience, nil); err != nil {
```

Comment:
We were incorrectly using the wrong flag (the one for test login). Audience is a required input for the test token command.